src/app/app.module.ts

```diff
@@ -20,11 +20,8 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { NotFoundComponent } from './components/not-found/not-found.component';
 import { LoginComponent } from './components/operations/login/login.component';
 import { RegisterComponent } from './components/operations/register/register.component';
+import { MyItinerariesComponent } from './components/itinerary/my-itineraries/my-itineraries.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181827)
```suggestion
import { MyItinerariesComponent } from "./components/itinerary/my-itineraries/my-itineraries.component";
```